test/evergreen.yml

```diff
@@ -1586,6 +1593,27 @@ tasks:
           display_name: Coverage report
           remote_file: wiredtiger/${build_variant}/${revision}/coverage_report/coverage_report_${build_id}.html
 
+  - name: wtperf-test
+    depends_on:
+      - name: compile-wtperf
+    commands:
+      - func: "fetch artifacts"
+        vars:
+          dependent_task: compile-wtperf
+      - command: shell.exec
+        params:
+          working_dir: "wiredtiger/build_posix"
+          script: |
+            set -o errexit
+            set -o verbose
+            # Python test suite generates WT_TEST directory
```

Comment:
This comment is a bit confusing here as we're not running Python test suite in this task.